jaqpotpy/api/jaqpot_api_client.py

```diff
@@ -193,7 +219,33 @@ def predict_with_csv_sync(self, model_id, csv_path):
             status_code=response.status_code.value,
         )
 
+    def get_dataset(self, response):
+        """
+        Retrieves a dataset using the location provided in the response headers.
+        Args:
+            response (requests.Response): The HTTP response object containing the dataset location in the headers.
```

Comment:
that's not true, the predict_async is returning the dataset location in the headers.

Reply:
`predict_async' returns the response and it is passed as input to `get_dataset`, so i don't get what is wrong here.

Reply:
Ah I see now you're passing the response to the get_dataset. If you want to make it easier for the user of the SDK maybe you can rename it to `get_dataset_from_prediction_response`. But still I don't think we should provide this to the user, instead we should put an example in the docs that we write this code. 

Remember the SDK should be "clean" in the sense that the users can use it for any purpose. We can't have util functions, cause they're not related with our SDK but we can have those in the docs